contracts/StakingVe.sol

```diff
@@ -0,0 +1,305 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
```

Comment:
This is activated by default in v0.8.0. maybe switch to that version and remove this.